.ci/pytorch/smoke_test/smoke_test.py

```diff
@@ -396,7 +396,9 @@ def smoke_test_nvshmem() -> None:
     except ImportError:
         # Not built with NVSHMEM support.
         # torch is not compiled with NVSHMEM prior to 2.9
-        if torch.__version__ < "2.9":
+        maj = int(torch.__version__.split(".")[0])
```

Comment:
Please don't do that (or at least use `maj, min = [int(x) for x in torch.__version__.split(".")[:2]]` ) but rely on TorchVersion